content/en/docs/tasks/traffic-management/egress/egress-tls-origination/index.md

```diff
@@ -194,17 +194,138 @@ are being sent by inspecting [Server Name Indication (SNI)](https://en.wikipedia
     $ kubectl delete destinationrule edition-cnn-com
     {{< /text >}}
 
-1.  Shutdown the [sleep]({{< github_tree >}}/samples/sleep) service:
+## Mutual TLS origination for egress traffic
+
+This section describes how to configure a sidecar to perform TLS origination for an external service, this time using a
+service that requires mutual TLS. This example is considerably more involved because it requires the following setup:
+
+1. Generate client and server certificates
+1. Deploy an external service that supports the mutual TLS protocol
+1. Configure the client (sleep pod) to use the credentials created in Step 1
+
+Once this setup is complete, you can then configure the external traffic to go through the sidecar which will perform
+TLS origination.
+
+### Generate client and server certificates and keys
+
+Follow [these steps](/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/#generate-client-and-server-certificates-and-keys)
+in the Egress Gateway TLS Origination task.
+
+### Deploy a mutual TLS server
+
+Follow [these steps](/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/#deploy-a-mutual-tls-server) in the Egress Gateway TLS Origination task.
+
+### Configure the client (sleep pod)
+
+1.  Create Kubernetes [Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) to hold the client's certificates:
+
+    {{< text bash >}}
+    $ kubectl create secret generic client-credential --from-file=tls.key=client.example.com.key \
+      --from-file=tls.crt=client.example.com.crt --from-file=ca.crt=example.com.crt
+    {{< /text >}}
+
+    The secret **must** be created in the same namespace as the client pod is deployed in, `default` in this case.
+
+1. Create required `RBAC` to make sure the secret created in the above step is accessible to the client pod, which is `sleep` in this case.
+
+    {{< text bash >}}
+    $ kubectl create role client-credential-role --resource=secret --verb=get,list,watch
+    $ kubectl create rolebinding client-credential-role-binding --role=client-credential-role --serviceaccount=default:sleep
+    {{< /text >}}
+
+### Configure mutual TLS origination for egress traffic at sidecar
+
+1.  Add a `DestinationRule` to perform mutual TLS origination
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: networking.istio.io/v1alpha3
+    kind: DestinationRule
+    metadata:
+      name: originate-mtls-for-nginx
+    spec:
+      workloadSelector:
+        matchLabels:
+          app: sleep
+      host: my-nginx.mesh-external.svc.cluster.local
+      trafficPolicy:
+        loadBalancer:
+          simple: ROUND_ROBIN
+        portLevelSettings:
+        - port:
+            number: 443
+          tls:
+            mode: MUTUAL
+            credentialName: client-credential # this must match the secret created earlier to hold client certs, and works only when DR has a workloadSelector
+            sni: my-nginx.mesh-external.svc.cluster.local
```

Comment:
sni is optional here.
you can remove it or add a comment here to avoid possible confusing.